core/src/main/resources/lib/form/textarea.jelly

```diff
@@ -105,7 +105,8 @@ THE SOFTWARE.
       <j:mute>${customizedFields.add(name)}</j:mute>
     </j:if>
   </f:possibleReadOnlyField>
-  <j:if test="${attrs.previewEndpoint!=null and !readOnlyMode}">
+
+  <j:if test="${attrs.previewEndpoint!=null and !readOnlyMode and app.markupFormatter.class.name != 'hudson.markup.EscapedMarkupFormatter'}">
```

Comment:
Am I missing something?
Nothing ties previews to markup formatters does it?  if so nothing here says the preview needs to be done using only a the Jenkins `MarkupFormatter`. 

```suggestion
  <j:if test="${attrs.previewEndpoint!=null and !readOnlyMode and (app.markupFormatter.class.name != 'hudson.markup.EscapedMarkupFormatter or attrs.previewEndpoint != '/markupFormatter/previewDescription')'}">
```

That is I could have a jenkins using the `EscapedMarkupFormatter` but I may have set the previewEndPoint to `MyDescriptorImpl.preview(String)`  

Indeed, the test on line 111 does imply that I should be able to use this irrespective of any global markup formatter.

a completely contrived example could be where a user supplies some toml, and my descriptor takes that toml (per job) and merges it with some global toml to create the "end toml" which is then displayed to the user?